Package.resolved

```diff
@@ -41,17 +41,17 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/braintree/braintree-ios-drop-in",
       "state" : {
-        "revision" : "cf6eead6b1751ff517d6100fa0c4d5c0a3628244",
-        "version" : "9.3.0"
+        "revision" : "81236a6c612f9b1ff04da6c12bc1869a859fe6fb",
+        "version" : "9.8.0"
```

Comment:
Do we want these braintree updates?

Reply:
you are right, attached by accident to PSP